docs(api): Fix mistakes in example code

alexjoel42 opened the pull request:
<!--
Thanks for taking the time to open a pull request! Please make sure you've read the "Opening Pull Requests" section of our Contributing Guide:

https://github.com/Opentrons/opentrons/blob/edge/CONTRIBUTING.md#opening-pull-requests

To ensure your code is reviewed quickly and thoroughly, please fill out the sections below to the best of your ability!
-->

# Overview

The [Pipette section of our API documentation ](https://docs.opentrons.com/v2/new_pipette.html#pipettes) had a lot of errors that would prevent users from simply copying and pasting code from the boxes. 




<!--
Use this section to describe your pull-request at a high level. If the PR addresses any open issues, please tag the issues here.
-->

# Changelog

- Updated API doc for correct load_instrument format
-Updated API doc for correct variable names 
- Updated API doc for syntax errors


<!--
List out the changes to the code in this PR. Please try your best to categorize your changes and describe what has changed and why.

Example changelog:
- Fixed app crash when trying to calibrate an illegal pipette
- Added state to API to track pipette usage
- Updated API docs to mention only two pipettes are supported

IMPORTANT: MAKE SURE ANY BREAKING CHANGES ARE PROPERLY COMMUNICATED
-->

# Review requests

- @ecormany  


<!--
Describe any requests for your reviewers here.
-->

# Risk assessment

There is a risk that I didn't catch everything or that there is a similar issue in other parts of our API documentation. 
<!--
Carefully go over your pull request and look at the other parts of the codebase it may affect. Look for the possibility, even if you think it's small, that your change may affect some other part of the system - for instance, changing return tip behavior in protocol may also change the behavior of labware calibration.

Identify the other parts of the system your codebase may affect, so that in addition to your own review and testing, other people who may not have the system internalized as much as you can focus their attention and testing there.
-->
## Overly detailed version of summary

1. Add a plate location [slot 4] in  line 74 and line 114. 
2. Fixed typos in load_instrument line 117-118. 
3. In the "Adding Tip Racks" section there is an issue where the tip rack referred to was "tiprack" and not "tiprack_left"  which caused a NameError. 
4. Line 278 the plate is accidentally defined as a tip rack. I named it as "corning_384_wellplate_112ul_flat" 
5. Line 345 (not established tip rack) line 384 plate is defined as a tiprack. Named as  "corning_384_wellplate_112ul_flat"
6. Move_To example needs to pick up a tip before moving

alexjoel42 committed:
Fixing Coding Errors in API documentation

I was making some Python API tutorial content and I realized that you can't copy and paste the code from our API documentation without getting errors.  


1. Add a plate location [slot 4] in  line 74 and line 114. 
2. Fixed typos in load_instrument line 117-118. 
3. In the "Adding Tip Racks" section there is an issue where the tip rack referred to was "tiprack" and not "tiprack_left"  which caused a NameError. 
4. Line 278 the plate is accidentally defined as a tip rack. I named it as "corning_384_wellplate_112ul_flat" 
5. Line 345 (not established tip rack) line 384 plate is defined as a tiprack. Named as  "corning_384_wellplate_112ul_flat"
6. Move_To example needs to pick up a tip before moving

ecormany commented on api/docs/v2/new_pipette.rst:
```diff
@@ -71,7 +71,7 @@ For instance, to aspirate from the first column of a 96-well plate you would wri
+        plate = protocol.load_labware('corning_96_wellplate_360ul_flat', 4)
 
         # Load a P300 Multi GEN2 on the right mount
         right = protocol.load_instrument(
```
still not runnable. needs to be `tip_racks=[tiprack1]` in line 78

alexjoel42 replied:
Changed!

ecormany commented on api/docs/v2/new_pipette.rst:
```diff
@@ -111,11 +111,11 @@ F, H, J, L, N, and P).
         # Load a P300 Multi GEN2 on the right mount
         right = protocol.load_instrument(
-            'p300_multi_gen2', 'right', tip_rack=tiprack1)
+            'p300_multi_gen2', 'right', tip_racks=[tiprack1])
```
this one is correct.

ecormany commented on api/docs/v2/new_pipette.rst:
```diff
@@ -391,7 +391,9 @@ the overall speed of the gantry. Its default is 400 mm/s.
-        pipette = protocol.load_instrument('p300_single', 'right')
+        tiprack = protocol.load_labware('opentrons_96_tiprack_300ul', '1')
+        pipette = protocol.load_instrument('p300_single', 'right', tip_racks = [tiprack])
+        pipette.pick_up_tip()
```
the original code runs, it just doesn't pick up a tip.

alexjoel42 replied:
Wowza, honestly I thought that we'd get a "tip not attached" error. I think you only get errors if you try and aspirate/dispense.  Thanks for catching this!

alexjoel42 replied:
I changed it back

ecormany replied:
yeah it's totally good either way

SyntaxColoring replied:
@alexjoel42 It looks like when changing this back, the `pipette.pick_up_tip()` line was accidentally retained.

With the added `pipette.pick_up_tip()` line, I believe this code will *not* run, because no list of tip racks has been defined for this pipette.

alexjoel42 replied:
Changed!

ecormany reviewed (changes requested):
one additional change needed on first example. the rest is valid and good to go.

alexjoel42 committed:
Following suggestions

@ecormany made some insightful recommendations:
1. missed tip rack definition
2. 394 was fine as is.

ecormany reviewed (approved):
good to go now!

alexjoel42 committed:
Fix additional pipette pick up and drop tip

@SyntaxColoring made a good point about this. Forgot to drop the pick_up_tip.
